### PR TITLE
Update 1.1.0 beta.3

### PR DESCRIPTION
**Pull request by vinnyjth**
### How do I test this PR?

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
🙌

File: packages/apollos-church-api/config.yml [hunk 1]
```diff
@@ -75,6 +77,7 @@ ROCK_MAPPINGS:
     UniversalContentItem:
       EntityType: ContentChannelItem
     WeekendContentItem:
+      ContentChannelId: [5]
```
Comment (redreceipt) [inline packages/apollos-church-api/config.yml]: is there a reason why this was added? I defined it already below.

Comment (redreceipt) [inline packages/apollos-church-api/config.yml]:
```suggestion
```

Comment (vinnyjth) [reply packages/apollos-church-api/config.yml]: Bad merge on my part

File: packages/apollos-church-api/package.json [hunk 2]
```diff
@@ -49,17 +50,18 @@
     "prettier": "1.12.1",
     "randomcolor": "0.5.3",
     "apollo-server-express": "2.4.8",
-    "express": "^4.17.0"
+    "express": "^4.17.0",
+    "@apollosproject/data-connector-cloudinary": "^1.1.0-beta.3"
```
Comment (redreceipt) [inline packages/apollos-church-api/package.json]: why is this added?

File: packages/newspringchurchapp/ios/newspringchurchappTests/Info.plist [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+`<?xml version="1.0" encoding="UTF-8"?>
```
Comment (redreceipt) [inline packages/newspringchurchapp/ios/newspringchurchappTests/Info.plist]: what about this???

File: packages/apollos-church-api/package.json (hunk 2 above)
Comment (vinnyjth) [reply packages/apollos-church-api/package.json]: Bad merge, again. You can remove if you are not using cloudinary.

File: packages/newspringchurchapp/ios/newspringchurchappTests/Info.plist (hunk 3 above)
Comment (vinnyjth) [reply packages/newspringchurchapp/ios/newspringchurchappTests/Info.plist]: wooooops

Comment (vinnyjth) [reply packages/newspringchurchapp/ios/newspringchurchappTests/Info.plist]: I did this when merging into your `develop` branch I think

File: packages/newspringchurchapp/package.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
   "version": "7.2.4",
   "private": true,
   "devDependencies": {
-    "@apollosproject/ui-storybook": "^1.1.0-beta.0",
+    "@apollosproject/ui-storybook": "^0.8.6",
```
Comment (redreceipt) [inline packages/newspringchurchapp/package.json]: core issue, dupe dependency?

Comment (vinnyjth) [reply packages/newspringchurchapp/package.json]: Yah, you can take it out from here

File: packages/apollos-church-api/package.json (hunk 2 above)
Comment (redreceipt) [inline packages/apollos-church-api/package.json]:
```suggestion
```

File: packages/newspringchurchapp/ios/newspringchurchappTests/Info.plist (hunk 3 above)
Comment (redreceipt) [inline packages/newspringchurchapp/ios/newspringchurchappTests/Info.plist]:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
```

File: packages/newspringchurchapp/package.json (hunk 4 above)
Comment (redreceipt) [inline packages/newspringchurchapp/package.json]:
```suggestion
```

**Comment by redreceipt** [comment]
we're seeing a 500 error when trying to query `userFeedFeatures`

Screen Shot 2019-08-06 at 1 51 26 PM

**Review by redreceipt — APPROVED** [review]
